NanoProd/crab/Run2_2016_HIPM/data.yaml

```diff
@@ -5,43 +5,50 @@ config:
     storeFailed: False
   #lumiMask: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/Legacy_2016/Cert_271036-284044_13TeV_Legacy2016_Collisions16_JSON.txt
 
-SingleElectron_Run2016B: /SingleElectron/Run2016B-ver2_HIPM_UL2016_MiniAODv2-v2/MINIAOD
+SingleElectron_Run2016Bv1: /SingleElectron/Run2016B-ver1_HIPM_UL2016_MiniAODv2-v2/MINIAOD
```

Comment:
here and below: As far as I know, there were no runs that are part of the golden json in `ver1`. What is the motivation to add them?

Reply:
That's correct, though I thought it could avoid some confusion if someone else who doesn't know about this wonders why 2018Bv1 isn't used when it's recommended by PdmV. If you think this won't be necessary, I can revert this to have just "2016B" pointing at v2.

Reply:
I think having a dataset with zero events that pass lumi preselection could be as confusing as not having it. Maybe you can add a comment to yaml about it for future references and revert to the previous version (only v2 and no need a v2 suffix in the name). Would you agree?